### DISCUSSION TITLE
Debug Menu Error

### DISCUSSION THREAD

**Discussion by Corpitell** [KadeArchive/Kade-Engine#550] [Q&A]
So I entered debug mode to try testing one of my custom characters but when I choose the custom character, i get an error saying Null Object Reference, its also happens to other game characters including when you head to the title menu sometimes, is there anything that I done wrong or what went wrong?

(Btw heres some screenshots of it and the cmd)

![Screenshot 2021-05-22 183644](https://user-images.githubusercontent.com/54646014/119242625-6eb1ca80-bb2d-11eb-9d83-1711ada60be7.png)
![Screenshot 2021-05-22 184202](https://user-images.githubusercontent.com/54646014/119242626-6f4a6100-bb2d-11eb-9146-d1b73460a977.png)
![Screenshot 2021-05-22 183615](https://user-images.githubusercontent.com/54646014/119242627-6f4a6100-bb2d-11eb-8c8b-a8b41eb4e0a4.png)

**Comment by pepperly-art** [comment]
The game's having an issue loading your character. Either you got a typo somewhere or you're missing files.

**Reply by Corpitell** [reply]
Oh alright, I will try to check the files to see if I made an error somewhere in the process of making the mod.

**Reply by Corpitell** [reply]
So apparently this happens with every version of the KadeDev engine but then I noticed that when I use gf in the debug menu in any song it dosent give me the error, but it may be because something inside the Character.hx file, but it may be the haxe, or the Microsoft VS Community applications, or the project settings HaxeDevelop, i will try one of those to see if that can fix it.
